src/styles/parts/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url('../images/star.svg');
+  }
+
+  &__star:first-child {
+    margin-left: 0;
+  }
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
```

Comment:
This code is repetitive. Consider using a loop to generate these classes. It will make your code more maintainable and easier to read.